NuKeeper.Abstractions/DTOs/UserPermissions.cs

```diff
@@ -0,0 +1,27 @@
+namespace NuKeeper.Abstractions.DTOs
+{
+    public class UserPermissions
+    {
+        public UserPermissions(bool admin, bool push, bool pull)
+        {
+            Admin = admin;
+            Push = push;
+            Pull = pull;
+        }
+
+        /// <summary>
```

Comment:
These comments are nice and descriptive on our DTO's. We can use them on the other properties for our DTO's as well.